app/assets/javascripts/editor/form.js.es6

```diff
@@ -25,6 +25,7 @@ $(() => {
     let $element = $(inserted[0]);
 
     applySelect2($element.find('select.select2'));
-    setupCreditOrder($element);
+    setupOrder($element, '.credit-order-value');
```

Comment:
Ah, right. Sorry for my unclear comment :<

This method name is pretty good!
Can you extract the abstract class name that means`'.credit-order-value` and `'.categorization-order-value'`?